From 03c04dc0fa0a3c35009cc4ea54c804b483076a07 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:11:41 -0400
Subject: [ethernet] i40evf: Remove unused defines

Message-id: <1409663591-32531-59-git-send-email-sassmann@redhat.com>
Patchwork-id: 90534
O-Subject: [RHEL7.1 PATCH 058/148] i40evf: Remove unused defines
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Catherine Sullivan <catherine.sullivan@intel.com>

Remove the defines for PCI bus info that are never used.

Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Sibai Li <sibai.li@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit dd8621a92b81e9d2bc4db5ee5b795c67cc04d3fa)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40e_type.h b/drivers/net/ethernet/intel/i40evf/i40e_type.h
index 51a6dee..a1055b4 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_type.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_type.h
@@ -101,15 +101,6 @@ enum i40e_debug_mask {
  I40E_DEBUG_ALL   = 0xFFFFFFFF
 };
 
-/* PCI Bus Info */
-#define I40E_PCI_LINK_WIDTH_1  0x10
-#define I40E_PCI_LINK_WIDTH_2  0x20
-#define I40E_PCI_LINK_WIDTH_4  0x40
-#define I40E_PCI_LINK_WIDTH_8  0x80
-#define I40E_PCI_LINK_SPEED_2500 0x1
-#define I40E_PCI_LINK_SPEED_5000 0x2
-#define I40E_PCI_LINK_SPEED_8000 0x3
-
 /* These are structs for managing the hardware information and the operations.
  * The structures of function pointers are filled out at init time when we
  * know for sure exactly which hardware we're working with.  This gives us the
-- 
1.7.1